.changeset/eighty-lies-hide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystone-next/admin-ui': major
+'@keystone-next/keystone': major
```

Comment:
I've made this a major for both packages because it's changing access so just in case someone is relying on the current default of not having a session meaning they're not allowed access to the Admin UI, it won't suddenly change in a patch